auth_pysaml/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2020 GlodoUK <https://www.glodo.uk/>
+# Copyright (C) 2010-2016 XCG Consulting <http://odoo.consulting>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Saml2 Authentication",
+    "version": "13.0.3.0.0",
+    "category": "Tools",
+    "author": "XCG Consulting, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/server-auth",
+    "license": "AGPL-3",
+    "depends": ["base_setup", "web"],
```

Comment:
`base_setup` already depends on `web`:

```suggestion
    "depends": ["base_setup"],
```